packages/opa/authz/rbac.rego

```diff
@@ -47,3 +69,75 @@ valid_params(expected, provided) if {
 		regex.match(sprintf("(?i)%s", [expected[k]]), provided[k])
 	}
 }
+
+handle_response(api_url, username, response) if {
+	# Print a log message if the response is not 200
+	response.status_code == 0
+	print("error connecting to server:", response)
+	false # regal ignore:constant-condition
+}
+
+handle_response(api_url, username, response) if {
+	# Print a log message if the response is not 200
+	response.status_code >= 300
+	print(
+		"ERROR: HTTP call to", api_url, "for user", username,
+		"FAILED. Status:", response.status_code, "Raw Body:", response.raw_body,
+	)
+	false # regal ignore:constant-condition
+}
+
+handle_response(api_url, username, response) if {
+	response.status_code == 200
```

Comment:
nothing between 201 and 299?

Reply:
It isn't needed, the automated-action API responds with `200` or an error code.